Frontend/src/components/pages/Project.jsx

```diff
@@ -0,0 +1,428 @@
+import {
+  Accordion,
+  AccordionSummary,
+  AccordionDetails,
+  Typography,
+  Container,
+  Button,
+  Link,
+  Box,
+  List,
+  ListItem,
+  ListItemIcon,
+  ListItemText,
+  IconButton,
+  CircularProgress,
+} from "@mui/material";
+import {
+  KeyboardArrowLeft as KeyboardArrowLeftIcon,
+  ExpandMore as ExpandMoreIcon,
+  Email as EmailIcon,
+  Phone as PhoneIcon,
+  Work as WorkIcon,
+  Description as DescriptionIcon,
+  AddCircle as AddCircleIcon,
+  Edit as EditIcon,
+  Delete as DeleteIcon,
+} from "@mui/icons-material/";
+
+import { useEffect, useState, useContext } from "react";
+import { useNavigate, useParams } from "react-router-dom";
+
+import ToastContext from "../../context/ToastContext";
+import DeleteAlertContext from "../../context/DeleteAlertContext";
+
+import CollaborationForm from "../forms/CollaborationForm";
+
+import SearchBar from "./partial/SearchBar";
+import TableComponent from "./partial/TableComponent";
+
+const tableColumns = [
+  {
+    key: "name",
+    label: "Company name",
+  },
+  {
+    key: "responsible",
+    label: "Responsible",
+    xsHide: true,
+  },
+  {
+    key: "status",
+    label: "Status",
+    centerContent: true,
+  },
+  {
+    key: "contact",
+    label: "Contact",
+    xsHide: true,
+  },
+  {
+    key: "category",
+    label: "Category",
+    notSortable: true,
+    centerContent: true,
+    xsHide: true,
+    mdHide: true,
+  },
+  {
+    key: "achievedValue",
+    label: "Value",
+    xsHide: true,
+  },
+  {
+    key: "comment",
+    label: "Comment",
+    xsHide: true,
+    mdHide: true,
+    showTooltip: true,
+  },
+];
+
+function handleEditTeamMember(e, id) {
+  // TODO: make a PUT request na /projects/{id}/teamMembers/{id} and then update frTeamMembers list
+  console.log("Editing a team member is not yet implemented!");
+  // setEditFormModal(true);
+}
+
+function handleDeleteTeamMember(e, id) {
+  // TODO: make a DELETE request na /projects/{id}/teamMembers/{id} and then update frTeamMembers list
```

Comment:
replace /projects/{id}/teamMembers/{id} with /projects/{id}

Reply:
Kako će znati kojeg će team membera maknuti?

Reply:
Na backend ce se poslat uz project object cijela nova lista project members bez tog membera kojeg smo maknuli kao da azuriramo sve podatke.

Reply:
Ne sviđa mi se to. Miješa funkcionalnosti i šalje previše stvari na backend.

Reply:
Sličnu razliku u mišljenjima imamo i u drugom PR.